setup.cfg

```diff
@@ -19,13 +19,15 @@ provides =
     phys2denoise
 
 [options]
-python_requires = >=3.6.1
+python_requires = >=3.9
 install_requires =
-    numpy >=1.9.3
-    matplotlib >=3.1.1
+    numpy >=1.9.3, <2
```

Comment:
Can you open an issue to state we need to refactor code to handle numpy >=2.0?

Reply:
Opened here: #62